bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -801,6 +801,36 @@ impl Room {
             }
         });
     }
+
+    pub fn get_timeline_event_content_by_event_id(
+        &self,
+        event_id: String,
+    ) -> Result<Arc<RoomMessageEventContent>, ClientError> {
+        let timeline = self
+            .timeline
+            .read()
+            .unwrap()
+            .clone()
+            .context("Timeline not set up, can't get event content")?;
+
+        let event_id = EventId::parse(event_id)?;
+
+        RUNTIME.block_on(async move {
+            let item = timeline
+                .item_by_event_id(&event_id)
+                .await
+                .context("Item with given event ID not found")?;
```

Comment:
As a small nit (that you can feel free to ignore), should we put only the `timeline.item_by_event_id(&event_id)` call into the `RUNTIME.block_on()`, so it's clearer that the rest of this method is sync?

Reply:
I don't really care much either way. Would really like to just use async methods :(

Reply:
> Would really like to just use async methods

What's stopping you? 🤔

Reply:
@stefanceriu That we still don't know why we sometimes get crashes with it! 😅

Reply:
Fair 😅 For what it's worth that seems limited to the timeline and the last message

Reply:
Will merge as-is because I don't think moving things out of `block_on` really changes much, and I don't want to add more stuff onto the CI queue 🥲